Detectors/ITSMFT/ITS/tracking/include/ITStracking/ClusterLines.h

```diff
@@ -209,11 +181,28 @@ GPUhdi() void Line::getDCAComponents(const Line& line, const float point[3], flo
   destArray[4] = std::sqrt(destArray[3] * destArray[3] + destArray[5] * destArray[5]);
 }
 
-///
+inline bool Line::operator==(const Line& rhs) const
+{
+  bool val;
+  for (int i{0}; i < 3; ++i) {
+    val = this->originPoint[i] != rhs.originPoint[i] ? false : true;
+  }
```

Comment:
shoud not this be:
```
  bool val = true;
  for (int i{0}; i < 3; ++i) {
    val &= this->originPoint[i] != rhs.originPoint[i] ? false : true;
  }
```

Reply:
or even just `val &= this->originPoint[i] == rhs.originPoint[i];`

Reply:
Thanks